.typos.toml

```diff
@@ -0,0 +1,29 @@
+# Configuration for https://github.com/crate-ci/typos
+
+[files]
+extend-exclude = [
+    "resources/linux-config-*",
+    "hypervisor/src/kvm/x86_64/mod.rs",
+]
+
+[default.extend-words]
+# External Interrupt
+EXTINT = "EXTINT"
+extint = "extint"
+INOUT = "INOUT"
+Ded = "Ded" # GUIDed
+ba = "ba"
+Cros = "Cros" # CrosVM
+Dake = "Dake"
```

Comment:
I wonder what's special about that name and not any of the others?

Reply:
![image](https://github.com/cloud-hypervisor/cloud-hypervisor/assets/5737016/3f8fc9f5-5b63-4aac-bf1a-b8a93f2eb3d0)

Reply:
You would think the tool would be able to identify that pretty common email format (`Name Name <email>`) and not flag it

Reply:
Agree, unfortunately it does not.